### ISSUE TITLE
Error while reading a message from server. {Value cannot be null. (Parameter 'paths')}

### ISSUE THREAD

**Issue by JoneKone** [Regalis11/Barotrauma#9473]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Can't join Dedicated server, I think it's due to over sized ban reason. (I accidentally pasted the server rules/Message of the day)

```
xxxx,STEAM_1:1:xxx99xxx9,,VIP's can join our filled servers! Pass the queue.

Support this server? Ask in discord, if you want to donate bottle of Cola. As this server is run by the power of Cola! And get a VIP, for your trouble.

Want to make changes and or contribute! Know, Lua or C#? join us! Discord link below.


Ru

Rule 1.2: Captains and Security cannot punish crew members unless they are disobeying achievable orders. Security is ALWAYS loyal to Captain! Captain and Security can NOT join a mutiny.
Rule 1.3: You may fight back if is in self defence the only exceptions to this is the use of husk eggs.
Rule 1.4: you can use explosive to dispatch enemies as long as you kill more enemies then innocent crew.

Rule 2: Mutiny needs at least 3 Members to declare.
Rule 2.1: Mutiny needs to be announced in the radio (!!text!!) chat before any hostile action.

Rule 4: Players behaving in a toxic manner will 
```

### Reproduction steps

1. Ban somebody with a wall of text copy paste the reason.
2. try to join the server.
not 100% sure this is the issue.

### Bug prevalence

Happens regularly

### Version

0.18.12.0

### Which operating system did you encounter this bug on?

Linux

### Relevant error messages and crash reports

```shell
Error while reading a message from server. {Value cannot be null. (Parameter 'paths')}
   at System.IO.Path.Combine(String[] paths)
   at Barotrauma.Networking.GameClient.ReadInitialUpdate(IReadMessage inc) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 1935
   at Barotrauma.Networking.GameClient.ReadLobbyUpdate(IReadMessage inc) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 2108
   at Barotrauma.Networking.GameClient.ReadDataMessage(IReadMessage inc) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 780
   at Barotrauma.Networking.LidgrenClientPeer.HandleDataMessage(NetIncomingMessage inc) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/Primitives/Peers/LidgrenClientPeer.cs:line 135
   at Barotrauma.Networking.LidgrenClientPeer.Update(Single deltaTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/Primitives/Peers/LidgrenClientPeer.cs:line 105
   at Barotrauma.Networking.GameClient.Update(Single deltaTime) in <DEV>/Barotrauma/BarotraumaClient/ClientSource/Networking/GameClient.cs:line 634
```

**Comment by JoneKone** [comment]
I removed the "bad" ban and it started to let ppl in.

**Comment by NilanthAnimosus** [comment]
Tested against dev commit https://github.com/Regalis11/Barotrauma-development/commit/d831645a3ccb4aadec718cb7146f0522c1ef4dae.


Initially I tried the dedicated server console and client console to give very long ban reasons (the one above copy pasted over itself even a few times) and there wasn't any visible issues, even if joining a dedicated after they were added. I had even tried copy pasting the above text over itself with no issues.

I figured perhaps its fine with the XML bans now, but when I went back to 0.18.15.1 I was unable to replicate the issue.
